CHANGELOG.md

```diff
@@ -3,67 +3,68 @@
+- **webdriverio:** include/exclude chaining and iframe selectors ([#409](https://github.com/dequelabs/axe-core-npm/issues/409)) ([ca8aa31](https://github.com/dequelabs/axe-core-npm/commit/ca8aa315251ae206d02843b125ee0e652258d186))
+- **webdriverio:** inject axe-core branding into all iframes ([#231](https://github.com/dequelabs/axe-core-npm/issues/231)) ([3f41c27](https://github.com/dequelabs/axe-core-npm/commit/3f41c2709114b894976bcdf03ca2b3d7f824fcae))
+- **webdriverio:** Recursively find <frame> ([#238](https://github.com/dequelabs/axe-core-npm/issues/238)) ([7e6a80d](https://github.com/dequelabs/axe-core-npm/commit/7e6a80d856118cceae598de5f91592d6e6c4dc39))
+- **webdriverio:** support wdio using puppeteer without esm ([#447](https://github.com/dequelabs/axe-core-npm/issues/447)) ([95dda29](https://github.com/dequelabs/axe-core-npm/commit/95dda2948e18035eaac4377ab9af6450005d0253))
+- **webdriverio:** use `executeAsync()` vs `execute()` ([#346](https://github.com/dequelabs/axe-core-npm/issues/346)) ([0e4aa3a](https://github.com/dequelabs/axe-core-npm/commit/0e4aa3ab6f26a48b70cabb7a5bd476e62658c951))
+- **webdriverJS:** include/exclude chaining and iframe selectors ([#404](https://github.com/dequelabs/axe-core-npm/issues/404)) ([c7c50fb](https://github.com/dequelabs/axe-core-npm/commit/c7c50fbe6ba91c51c3693ac1220fbd6470532a88))
+- **webdriverjs:** prevent selnium undefined -> null transformation ([#402](https://github.com/dequelabs/axe-core-npm/issues/402)) ([5095f43](https://github.com/dequelabs/axe-core-npm/commit/5095f43d371a3ad5c8b5a6b3f94e0ad686e85d7b))
+- **webdriverjs:** prevent selnium undefined -> null transformation ([#402](https://github.com/dequelabs/axe-core-npm/issues/402)) ([be3912d](https://github.com/dequelabs/axe-core-npm/commit/be3912d47f6a9d5507aec6af2a01484de554daec))
+- **webdriverjs:** Recursively find <frame> ([#209](https://github.com/dequelabs/axe-core-npm/issues/209)) ([0d20e1f](https://github.com/dequelabs/axe-core-npm/commit/0d20e1f3238ca70ccb528867fb1456487c02d0df))
+- **webdriverjs:** Reject with actual `Error`s (not strings) ([#423](https://github.com/dequelabs/axe-core-npm/issues/423)) ([3fdb50a](https://github.com/dequelabs/axe-core-npm/commit/3fdb50ad7b9106fa288d7c2b3092ec31de5d984b)), closes [#422](https://github.com/dequelabs/axe-core-npm/issues/422) [#421](https://github.com/dequelabs/axe-core-npm/issues/421) [#387](https://github.com/dequelabs/axe-core-npm/issues/387) [#308](https://github.com/dequelabs/axe-core-npm/issues/308) [#207](https://github.com/dequelabs/axe-core-npm/issues/207)
+- **webdriverjs:** resolve promise ([#347](https://github.com/dequelabs/axe-core-npm/issues/347)) ([d1548a5](https://github.com/dequelabs/axe-core-npm/commit/d1548a5ad8c31262a655b7ba1e4fe5b7da888417))
 
+### Features
```

Comment:
What is going on here? I seriously doubt you bumped axe-core versions multiple times in this patch release. Something's wrong with how these logs are getting generated.

Reply:
It looks like these logs are just getting pushed down and are denoted as "new" changelogs

Reply:
Aren't these changelog files automatically generated? I'm not sure what we can do about it.

What exactly are the changes you're requesting here?

Reply:
If the changelogs are incorrect my guess is it's the same problem we had in axe-core. When the changelog is generated it looks for the last tagged commit on your machine. Since our commits are tagged by the release ci process, most of the time our local machines don't have up-to-date tags. We fixed that by forcing the download of the latest tags before creating the release

https://github.com/dequelabs/axe-core/blob/develop/package.json#L103. Also add the `skip: tag` option to `standard-commit` https://github.com/dequelabs/axe-core/blob/develop/package.json#L64-L66

Reply:
This repository uses Lerna. Not sure the `skip: tag` thing applies.

Having the script fetch tags before releasing is a great idea. I'm going to steal that 😄